media-gfx/graphviz/graphviz-2.40.1.recipe

```diff
@@ -0,0 +1,165 @@
+SUMMARY="Open source graph visualization software"
+DESCRIPTION="Graphviz is open source graph visualization software. Graph \
+visualization is a way of representing structural information as diagrams of \
+abstract graphs and networks. It has important applications in networking, \
+bioinformatics,  software engineering, database and web design, machine \
+learning, and in visual interfaces for other technical domains.
+The Graphviz layout programs take descriptions of graphs in a simple text \
+language, and make diagrams in useful formats, such as images and SVG for web \
+pages, PDF or Postscript for inclusion in other documents; or display in an \
+interactive graph browser. (Graphviz also supports GXL, an XML dialect.)
+Graphviz has many useful features for concrete diagrams, such as options for \
+colors, fonts, tabular node layouts, line styles, hyperlinks, rolland custom \
+shapes."
+HOMEPAGE="http://www.graphviz.org/"
+COPYRIGHT="1994-2010 AT&T Corp."
+LICENSE="CPL v1.0"
```

Comment:
The license is now [named `EPL v1.0`][0]

[0]: https://gitlab.com/graphviz/graphviz/blob/stable_release_2.40.1/LICENSE#L1

Reply:
https://www.graphviz.org/license/ (this one still calls it CPL v1.0)?